sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py

```diff
@@ -165,4 +165,52 @@ def test_content_continuation_token(self, resource_group, location, form_recogni
         poller = client.begin_recognize_content_from_url(self.form_url_jpg, continuation_token=cont_token)
         result = poller.result()
         self.assertIsNotNone(result)
-        initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
\ No newline at end of file
+        initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    def test_content_multipage_table_span_pdf(self, resource_group, location, form_recognizer_account, form_recognizer_account_key):
+        client = FormRecognizerClient(form_recognizer_account,
+                                      AzureKeyCredential(form_recognizer_account_key))
+
+        poller = client.begin_recognize_content_from_url(self.multipage_table_url_pdf)
+        result = poller.result()
+        self.assertEqual(len(result), 2)
+        layout = result[0]
+        self.assertEqual(layout.page_number, 1)
+        self.assertEqual(len(layout.tables), 2)
+        self.assertEqual(layout.tables[0].row_count, 30)
+        self.assertEqual(layout.tables[0].column_count, 5)
+        self.assertEqual(layout.tables[0].page_number, 1)
+        self.assertEqual(layout.tables[1].row_count, 6)
+        self.assertEqual(layout.tables[1].column_count, 5)
```

Comment:
also would it be better to have a helper function that compares two form pages

Reply:
done in the transform test